src/js/transports/TextEncoder.ts

```diff
@@ -0,0 +1,13 @@
+import { TextEncoderInternal } from '@sentry/types';
+import { Buffer } from 'buffer';
+
+export const makeUtf8TextEncoder = (): TextEncoderInternal => {
+  const textEncoder = {
+    encode: (text: string) => {
+      const bytes = new Uint8Array(Buffer.from(text, 'utf8'));
```

Comment:
Let's try to avoid deps and vendor the class from `buffer` if possible.
https://develop.sentry.dev/sdk/philosophy/#dependencies-cost